README.md

```diff
@@ -295,6 +297,27 @@ To reduce the size of the installation from about 800 MByte down to about 200 MB
 * Build error around `prebuild: npm run bootstrap`
    * This has been seen failing on WSL.  Try using Node 12.13.0
 
+### As a Docker image
+
+Alternatively, Node-wot can be built as a Docker image with the `Dockerfile`.
```

Comment:
Picky, I know .. may I ask you to lower-case the "N" for node-wot

Reply:
Oh yeah, thanks!
Should I also change "Node-wot" at the beginning of https://github.com/eclipse/thingweb.node-wot/blob/f2c06ab36463bfc8009a830b805b76b7d2a289e3/README.md#as-a-browser-library or is it fine there as it starts the sentence?

Reply:
I stumbled over this as well... I did not mention it since it is in the beginning of the sentence but YES, I think we should change that as well. Thanks!